Code/Tools/SceneAPI/SceneBuilder/SceneSystem.cpp

```diff
@@ -28,6 +29,13 @@ namespace AZ
 
         void SceneSystem::Set(const SDKScene::SceneWrapperBase* scene)
         {
+            static constexpr const char* s_UseOldAssimpBehaviorKey = "/O3DE/Preferences/SceneAPI/UseOldAssimpBehavior";
+            bool useOldAssimpBehavior;
+            if (!AZ::SettingsRegistryInterface::Instance()->Get(useOldAssimpBehavior, s_UseOldAssimpBehaviorKey))
+            {
+                // Default to true to maintain old behavior.
+                useOldAssimpBehavior = true;
+            }
```

Comment:
For instance, negating would simplify this to 
```suggestion
            bool useNewAssimpBehavior{ false };
            AZ::SettingsRegistryInterface::Instance()->Get(useNewAssimpBehavior, s_UseNewAssimpBehaviorKey))
```